hs_csdms/utils.py

```diff
@@ -0,0 +1,31 @@
+def split_csdms_names():
+    """ Split each record in csdms file and assign them to corresponding return part
+        :return names, names in csdms
+        :return splitted_names, split each name by space
+        :return decors, decorations for each name in csdms
+        :return splitted_decors, split each decor by space
```

Comment:
you have 4 returns here but only 3 are being returned.  Names are also inconsistent.

Reply:
fixed